examples/terraform/azure/output.tf

```diff
@@ -47,7 +47,7 @@ output "kubeone_workers" {
     # following outputs will be parsed by kubeone and automatically merged into
     # corresponding (by name) worker definition
     "${var.cluster_name}-pool1" = {
-      replicas = 1
+      replicas = var.worker_node_vm_count
```

Comment:
This should also be changed to `initial_machinedeployment_replicas`.

Reply:
Done